src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawing {
+    private String colorfigure;
+    private double calcutateArea;
```

Comment:
we shouldn't save figure area as a field
```suggestion
    private String color;
```